README.md

```diff
@@ -69,6 +72,11 @@ insertIssueLinkInPrDescription:
     docOnlyIssueMatch:
       titleIssueIdRegexp: \[(AIRFLOW-X{4})\]
       descriptionIssueLink: "`Document only change, no JIRA issue`"
+
+# Verifies if all commit subjects match the regexp specified
+verifyCommitMessages:
+  # Regular expression that should be matched by subject of each commit in the PR
+  commitMessageRegexp: ^.*\[AIRFLOW-[0-9]{4}\].*$|^.*\[AIRFLOW-[X]{4}\].*$
```

Comment:
```suggestion
  commitMessageRegexp: ^.*\[AIRFLOW-[0-9]{4}\].*$|^.*\[AIRFLOW-X{4}\].*$
```
Some nit here.

Reply:
BTW, should we fix the `AIRFLOW-[0-9]{4}` OR `ARIFLOW-XXXX` in the beginning?
```suggestion
  commitMessageRegexp: ^\[AIRFLOW-[0-9]{4}\].*$|^\[AIRFLOW-X{4}\].*$
```

Reply:
And maybe someone will create PR for an old Jira ticket, so we should use `{1,4}` 
```suggestion
  commitMessageRegexp: ^\[AIRFLOW-\d{1,4}\].*$|^\[AIRFLOW-X{4}\].*$
```

Reply:
I updated a few things now (pointed out by @nuclearpinguin and @mik-laj)
1) the squashed commit can come either from PR title (if you have multiple commits) or from commit (if there is a single commit). So I added a configuration option to check exactly those.

2) @zhongjiajie -> I think we should not worry about old JIRAs. We can always merge those even if we see that commit validation fails (committers can do it). I think there is a value in someone not making an accidental copy&paste without last digit.